v0.11.0/README.md

```diff
@@ -0,0 +1,75 @@
+![](/docs/img/velero.png)
+
+[![Build Status][1]][2] <a href="https://zenhub.com"><img src="https://raw.githubusercontent.com/ZenHubIO/support/master/zenhub-badge.png"></a>
+
+## Heptio Ark is now Velero!
+
+#### We're working on our first Velero release and instructions for migrating your Ark deployments to Velero. Stay tuned!
```

Comment:
I think we can remove this line with this PR, since this PR includes the instructions for migration.

Reply:
FYI PR #1203 removes this (from `master`) -- ideally gets merged right before release.

Reply:
That would work, though this will need to be regenerated after that change is merged.